packages/chain-adapters/src/index.ts

```diff
@@ -1,6 +1,6 @@
 export * from './api'
 export * from './utils'
-export * as chainAdapters from './types'
+export * from './types'
```

Comment:
If we want this not to be a breaking change, we should have both the previous line (exporting all the modules as a `chainAdapters` export) and this new one (exporting all exports from this module).

Reply:
My vote is update the description and carry on seeing as how the destructured import is not being used in web yet.

Reply:
Agree. Updated accordingly.